src/FSharpSource.targets

```diff
@@ -102,7 +102,7 @@
       <PropertyGroup >
         <OutputPath>$(FSharpSourcesRoot)\..\$(Configuration)\$(ProtoFlavour)\bin</OutputPath>
         <!-- When using coreclr to bootstrap to proto. We acquire the FSharp.Core LKG, targets LKG, the FSharp.Build LKG from the .NET Framework compiler tools LKG package. -->
-        <FSharpNetCoreLkgPath>$(FSharpSourcesRoot)\..\Tools\dotnet20\sdk\2.0.3-servicing-007056\FSharp</FSharpNetCoreLkgPath>
```

Comment:
Read this value from the file so we don't have to update it again.  Something like the following should work:

``` xml
<DotnetSdkVersion>$([System.IO.File]::ReadAllText('..\DotnetCLIToolsVersion.txt').Trim())</DotnetSdkVersion>
<FSharpNetCoreLkgPath>$(FSharpSourcesRoot)\..\Tools\dotnet20\sdk\$(DotnetSdkVersion)\FSharp</FSharpNetCoreLkgPath>
```